api/proto/banyandb/database/v1/rpc.proto

```diff
@@ -64,11 +70,33 @@ message StreamRegistryServiceListResponse {
 }
 
 service StreamRegistryService {
-  rpc Create(StreamRegistryServiceCreateRequest) returns (StreamRegistryServiceCreateResponse);
-  rpc Update(StreamRegistryServiceUpdateRequest) returns (StreamRegistryServiceUpdateResponse);
-  rpc Delete(StreamRegistryServiceDeleteRequest) returns (StreamRegistryServiceDeleteResponse);
-  rpc Get(StreamRegistryServiceGetRequest) returns (StreamRegistryServiceGetResponse);
-  rpc List(StreamRegistryServiceListRequest) returns (StreamRegistryServiceListResponse);
+  rpc Create(StreamRegistryServiceCreateRequest) returns (StreamRegistryServiceCreateResponse){
+    option(google.api.http) = {
+      post:"/v1/stream/schema"
+      body:"*"
+    };
+  };
+  rpc Update(StreamRegistryServiceUpdateRequest) returns (StreamRegistryServiceUpdateResponse){
+    option(google.api.http) = {
+      put:"/v1/stream/schema"
```

Comment:
"/v1/stream/schema/{metadata.group}/{metadata.name}"

Reply:
This patch is good for me.